src/js/notific8.js

```diff
@@ -170,6 +170,18 @@ notific8 = (function() {
     notification = notification.replace('$notificationClasses', generatedNotificationClasses.join(' '));
     container.innerHTML += notification;
 
+    if (typeof data.settings.heading !== 'undefined') {
+        let currentNotificationHeading = container.getElementsByClassName(`${data.settings.namespace}-heading`);
```

Comment:
This looks good. I'm just wondering if there is a case where we'd need to worry about multiple elements matching this query string? I can't think of any but I might be missing some. If not, would it make sense to reduce this function and the next down to just two lines something like the following?

```js
const currentNotificationHeading = container.querySelector(`.${data.settings.namespace}-heading`);
currentNotificationHeading.textContent = `${data.settings.heading}`;
```

Either way, since we're not mutating the values of the variables being set, we should probably make them `const` instead of `let`. It may sound silly but I know being vigilant about that has saved me from myself at work loads of times.

Reply:
That would be way better, but it would cause an issue and that is when you spam the notification you would override the first found element and the new notification would be empty strings, so the `three lines` one makes sure you will get the latest notification. Still I am sure there are better ways and I might re-write things when I'm free.

Reply:
If there is a unique id (generated identifier) for each notification container the code you suggested would work easily.

Reply:
That's a good point.